TUM Campus App/CardTableViewCell.swift

```diff
@@ -7,12 +7,30 @@
 //
 
 import UIKit
-import MCSwipeTableViewCell
-
-class CardTableViewCell: MCSwipeTableViewCell {
-    
-    func setElement(_ element: DataElement) {
-        
-    }
 
+@IBDesignable class CardTableViewCell: UITableViewCell {
+	
+	@IBInspectable var color1: UIColor = UIColor.white
+	@IBInspectable var color2: UIColor = UIColor.black
```

Comment:
`color1` and `color2` seem to be quite non-descript. Is there perhaps a better way to express what colour is used for what part of the UI?